sounddevice.py

```diff
@@ -2776,20 +2803,27 @@ def _split(value):
     return invalue, outvalue
 
 
-def _check(err, msg=""):
+def _check(err, msg=None):
```

Comment:
I also don't think it'll be an issue here, for me it's also about good practice.

In your version, I get exactly the awkward output you mentioned if I use `_check(xxx, '')` (which of course I wouldn't do in practice, but still).

I think it's more reasonable (and more Pythonic, too) to ignore it if it is *falseish*.

This is IMHO very idiomatic:

```python
if msg:
    # do something
```

I would use `None` only if there are valid falseish values which I care about (and if the default value is something mutable, which might be problematic in some cases).